mailer-quickstart/src/main/java/org/acme/getting/started/MailerResource.java

```diff
@@ -35,6 +35,7 @@
                 .to(email)
                 .subject("Ahoy " + name + "!")
                 .send()
+                .subscribeAsCompletionStage()
```

Comment:
It would be better to change the signature and return a Uni. This is the model we recommend.

Reply:
So the docs need update too then?
https://quarkus.io/guides/mailer - `public CompletionStage<Response>`

Reply:
yes